app/src/renderer/components/common/TmSessionImport.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="tm-session">
-    <tm-form-struct :submit="onSubmit" class="tm-session-container">
+    <tm-form-struct :submit="onSubmit.bind(this)" class="tm-session-container">
```

Comment:
I'm not 100% sure it will work, let's try to deconstruct from ({$v, $store...} = this) => () => {actual stuff}

Reply:
Works, you tested it yourself ;)